include/wlr/render/egl.h

```diff
@@ -9,6 +9,10 @@
 #ifndef WLR_RENDER_EGL_H
 #define WLR_RENDER_EGL_H
 
+#ifndef WLR_HAS_X11_BACKEND && WLR_HAS_XWAYLAND
```

Comment:
This isn't how `ifdef/ifndef` works. You want something more like `#if defined(...) && defined(...)`.

Reply:
Fixed. Thank you!